modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -857,6 +859,15 @@ private void _addObjectDefinitionResources(
 						postObjectDefinitionObjectRelationship(
 							objectDefinitionId, objectRelationship);
 
+				_objectRelationshipLocalService.updateEdge(
+					objectRelationshipEdge, objectRelationship.getId());
+
+				ObjectField relationshipObjectField =
+					objectRelationship.getObjectField();
+
+				_objectFieldLocalService.updateRequired(
+					relationshipObjectField.getId(), true);
```

Comment:
@G-Sa this commit is not a `buildService` 🥲 

![Alt Text](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExeWw2dDFrOHg5MDkzYXE3dm1xd3ZnZG8xcDhzZG8weGxtMDB5ZjkxMSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/xT5P0xr3oUJM1xNH5S/giphy.gif)